app/uk/gov/hmrc/soletraderidentificationfrontend/featureswitch/core/config/FeatureSwitchingModule.scala

```diff
@@ -52,4 +52,14 @@ case object KnownFactsStub extends FeatureSwitch {
 case object EnableNoNinoJourney extends FeatureSwitch {
   override val configName: String = "feature-switch.enable-no-nino-journey"
   override val displayName: String = "Enable no nino journey"
+}
+
+case object EnableNinoIVJourney extends FeatureSwitch {
+  override val configName: String = "feature-switch.enable-nino-iv-journey"
+  override val displayName: String = "Enable Nino IV Journey"
```

Comment:
Looking at other switches for consistency purpose, can we do "Enable Nino IV journey" (J in lowercase)